_data/projects/i-can-not-react.yml

```diff
@@ -0,0 +1,13 @@
+name: i-can-not-react
+desc: "The Complete Book on Reactjs"
+site: "https://github.com/KirankumarAmbati/I-can-not-REACT"
```

Comment:
@KirankumarAmbati I guess the reason for failing travis are these '"'. Please remove these from line 2 and line 3 both.